docs/credits.rst

```diff
@@ -258,7 +258,7 @@ Other Credits
 * Everyone on the `astropy-dev mailing list`_ and the `Astropy mailing list`_
   for contributing to many discussions and decisions!
 
-(If you have contributed to the Astropy project and your name is missing,
+(If you have contributed to the Astropy Project and your name is missing,
```

Comment:
I'm not sure it should be astropy project here, as thsi is the docs of the core package. But probably we just talk about a very small number of people who contributed to the project but not to core....